sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/AddressResolverTest.java

```diff
@@ -216,7 +216,7 @@ private void TestCacheRefreshWhileRouteByPartitionKey(
         try {
             resolvedAddresses = this.addressResolver.resolveAsync(request, forceAddressRefresh).block();
         } catch (RuntimeException e) {
-            throw (Exception) e.getCause();
+            throw e;
```

Comment:
as this is test not sure it matters or not. but this is a logic change no?

Reply:
Reverted